### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qu1queee**
[#169619003](https://www.pivotaltracker.com/n/projects/2192232/stories/169619003)

This add a list of tools that can be helpful
when using this repository and kubecf.
Also, fix some YAML lint warnings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +143,45 @@ For more information about testing, see [docs/testing.md](docs/testing.md)
 For more information about building the operator from source, see [docs/building.md](docs/building.md)
 
 For more information about how to develop a BOSH release using Quarks and SCF, see the [SCFv3 docs](https://github.com/SUSE/scf/blob/v3-develop/dev/scf/docs/bosh-author.md)
+
+## Nice Tools to use
```
Comment (edwardstudy) [inline README.md]: :Lower case `tools`?

File: README.md [hunk 2]
```diff
@@ -143,3 +143,45 @@ For more information about testing, see [docs/testing.md](docs/testing.md)
...
+
+The following is a list of tools with their respective main features that can help you
+to simplify your development work when dealing with [cf-operator](https://github.com/cloudfoundry-incubator/cf-operator) and [kubecf](https://github.com/SUSE/kubecf)
+
+### [k9s](https://github.com/derailed/k9s)
+
+Provides an easy way to navigate through your k8s resources, while watching lively
```
Comment (edwardstudy) [inline README.md]: Could we add a it before `provides`?

File: README.md (hunk 1 above)
Comment (qu1queee) [reply README.md]: yes

File: README.md (hunk 2 above)
Comment (qu1queee) [reply README.md]: yes

**Comment by qu1queee** [comment]
@edwardstudy I modified per your request

**Pull request merged**